### PR TITLE
neogeo - dumped latest Japan bios from MV1C PCB [Razoola]

### PR DESCRIPTION
**Pull request by DavidHaywood**
reorganized BIOS roms a bit, correcting some names, making primary bios from a known board type etc.

### CODE REVIEW

File: src/mame/drivers/neogeo.cpp [hunk 1]
```diff
@@ -3219,7 +3239,7 @@ ROM_START( ridheroh )
 	/* Chip label p1h does not exist, renamed temporarly to pg1, marked BAD_DUMP. This needs to be verified. */
 
 	ROM_REGION( 0x2000, "mcu", 0 )    /* Hitachi HD6301V1 MCU */
-	ROM_LOAD( "rhcom.bin", 0x0000, 0x2000, CRC(e5cd6306) SHA1(f6bbb8ae562804d67e137290c765c3589fa334c0) ) // dumped from a prototype with external ROM, not 100% confirmed as being the same on a final, or other games (lbowling, trally)
+	ROM_LOAD( "rhcom.bin", 0x0000, 0x2000, CRC(e5cd6306) SHA1(f6bbb8ae562804d67e137290c765c3589fa334c0) )
```
Comment (cuavas) [inline src/mame/drivers/neogeo.cpp]: Did we get confirmation that the internal ROMs match for these?  That's pretty cool if we did.

**Comment by DavidHaywood** [comment]
The MV-1C bios is the latest revision tho, which is usually what policy dictates is the parent.

File: src/mame/drivers/neogeo.cpp (hunk 1 above)
Comment (DavidHaywood) [reply src/mame/drivers/neogeo.cpp]: the internal ROM isn't confirmed, the external roms for everything else on the prototype did 100% match final tho, as per the comment change, so it's very likely these match for this game.  the only change here is moving the comment higher up.

**Comment by cuavas** [comment]
Yes, but it's for a machine we don't really emulate yet - we only emulate variants with an onboard M1 ROM.

**Comment by DavidHaywood** [comment]
well I'll close this and let somebody else take care of it then

**Pull request closed without merging**